docs/agent_hocon_reference.md

```diff
@@ -412,19 +394,19 @@ This allows common agent network definitions to be used as functions for other l
 
 Furthermore, it is also possible to reference agents on other neuro-san _servers_ by using a URL as a tool reference.
 
-Example: "http://localhost:8080/math_guy"
+Example: [http://localhost:8080/math_guy](http://localhost:8080/math_guy)
 
 This enables entire ecosystems of agent webs.
 
-### llm_config
+### Per agent llm_config
```

Comment:
What tool are you using that is motivating this change?
I don't like that it's inconsistent with all the other keys?

Reply:
markdownlinter-cli